server/RoboFile.php

```diff
@@ -146,7 +146,13 @@ public function deployPantheon($branchName = 'master') {
    * @throws \Robo\Exception\TaskException
    */
   public function deployPantheonSync(string $env = 'test', bool $doDeploy = TRUE) {
-    $pantheonName = self::PANTHEON_NAME;
+    if (getenv('PANTHEON_NAME')) {
```

Comment:
Shall we document this somewhere?

Reply:
@AronNovak I we can add it to https://github.com/Gizra/ihangane/wiki/Deployment#project-specific-scripts
We also need `GITHUB_USERNAME` and `GITHUB_ACCESS_TOKEN` for the deploy, so, maybe we should add few words on it as well.